### PR TITLE
Adding an event listener in case the chat window is hidden for notifications

### PR DESCRIPTION
**Pull request by dishakhattri**
- Adding support in case the chat window is hidden

### CODE REVIEW

File: CHANGE_LOG.md [hunk 1]
```diff
@@ -97,3 +97,4 @@ All notable changes to this project will be documented in this file.## [Unreleas
 - Fixed PostChat loading twice when Bot enabled
 - Adding `hideErrorUIPane` to `ILiveChatWidgetControlProps` to show/hide Error UI on start chat
 - Fix Start Chat icon disappearing due to subtitle length
+- Adding support in case the chat window is hidden
```
Comment (charliewang95) [inline CHANGE_LOG.md:100]: Please elaborate more on this note

File: docs/Telemetry.md [hunk 2]
```diff
@@ -138,6 +138,9 @@ Refer to the below table to understand different broadcast events raised during
 | `InitiateEndChatOnBrowserUnload`      | End active chats on browser unload |
 | `ClosePopoutWindow`      | Event to close popout window  |
 | `RaiseErrorEvent`                 | On raising error events |
+| `NetworkDisconnected`                 | On network Disconnects |
+| `NetworkReconnected`                 | On network reconnects |
+| `HideChatVisibilityChangeEvent`                 | On hiding the chat window or tab |
```
Comment (charliewang95) [inline docs/Telemetry.md:143]: This is not something chat-widget send, but receives. We don't need to log it here

**Review by charliewang95 — APPROVED** [review]

File: chat-widget/src/common/telemetry/TelemetryConstants.ts [hunk 3]
```diff
@@ -52,6 +52,7 @@ export enum BroadcastEvent {
     SigninCardReceived = "SignInCardReceived",
     BotAuthConfigRequest = "BotAuthConfigRequest",
     BotAuthConfigResponse = "BotAuthConfigResponse",
+    HideChatVisibilityChangeEvent = "hideChatVisibilityChangeEvent"
```
Comment (marghayk) [inline chat-widget/src/common/telemetry/TelemetryConstants.ts:55]: nit: Upper case for consistency?

**Review by marghayk — APPROVED** [review]

File: chat-widget/src/common/telemetry/TelemetryConstants.ts (hunk 3 above)
Comment (marghayk) [reply chat-widget/src/common/telemetry/TelemetryConstants.ts:55]: please disregard, it looks like incoming event is lower case too

**Pull request merged**